heron/statemgrs/src/java/com/twitter/heron/statemgr/zookeeper/curator/CuratorStateManager.java

```diff
@@ -310,4 +320,19 @@ public void processResult(CuratorFramework aClient, CuratorEvent event) throws E
       return deleteNode(getSchedulerLocationPath(topologyName));
     }
   }
+
```

Comment:
Instead of having the main there - could we separate this into tool called - 

```
heron-zk 
```
that can be used to show the contents of state managers -  this tool can take the name of the cluster and use the appropriate config file for state manager and display the contents - will be useful for troubleshooting?